README.md

```diff
@@ -58,3 +58,188 @@ select * where {
     <urn:a> ?p ?o  .
 }
 ```
+
+# Secured Kafka Sink with FailedRecordProducer Configuration
+
+This guide provides instructions on how to start a secured Kafka Sink while configuring
+
+the inner FailedRecordProducer with security properties. The configuration can be provided
+
+either using properties with a "producer.override." prefix or through environment variables in the Dockerfile.
+
+## Configuration Options
+
+### Using "producer.override." Prefix in Properties
+
+To configure the inner FailedRecordProducer with security properties using a "producer.override." prefix:
+
+Provide the `graphdb-kafka-sink.properties` file to include security properties with the "producer.override." prefix. For example:
+
+```properties
```

Comment:
Aren't those passed when the connector is created? If so maybe a sample connector config would be better.

Reply:
There are three way for starting connector and I'm giving example for each